City-Bureau/city-scrapers-cle
# 🕷️ Fix spider: Cuyahoga County Personnel Review Commission

## What's this PR do?

Rebuilds our Cuyahoga County Personnel Review Commission spider (aka. `cuya_personnel_review_commission`).

## Why are we doing this?

The spider broke long ago and was deprecated in #93. Our Cleveland partners wanted to see the scraper rebuilt.

## Steps to manually test

After installing the project using `pipenv`:

1. Activate the virtual environment:
```
pipenv shell
```
2. Run the spider:
```
scrapy crawl cuya_personnel_review_commission -O test_output.csv
```
3. Monitor the stdout and ensure that the crawl proceeds without raising any errors. Pay attention to the final status report from scrapy.

4. Inspect `test_output.csv` to ensure the data looks valid. I suggest opening a few of the URLs under the source column of test_output.csv and comparing the data for the row with what you see on the page.

## Are there any smells or added technical debt to note?
